### PR TITLE
PSMDB-1508 Release notes 5.0.28-24

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   .cache/plugin/git-committers/page-authors.json
	modified:   .github/workflows/main.yml
	new file:   docs/release_notes/5.0.28-24.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   requirements.txt
	modified:   variables.yml

### CODE REVIEW

**Comment by radoslawszulgo** [comment]
Can we highlight each section in highlights even more? 
For example, each highlight item should get a dedicated h3 heading. Each header states a value for the user. For example:

### Better product usage reporting with enhanced telemetry
This release of Percona Server for MongoDB includes ...

### Reduce mean time to resolve compromised encryption keys incident.
....

### Easier dependency management with thinner tarballs
...

File: docs/release_notes/5.0.28-24.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Starting with this release, Percona Server for MongoDB automatically activates the master encryption keys at startup and periodically checks (polls) their status in a KMIP server. If a master encryption key for a node is not in the Active state, the node reports an error and shuts down. This method allows security engineers to quickly identify which nodes require out-of-schedule master key rotation, such as in the case of compromised keys, without needing to rotate keys for the entire cluster. 
+
+Learn more about key state polling from [documentation](../kmip.md#key-state-polling) 
+
+### Easier dependency management with thinner tarballs 
+
+Tarballs are now available for each supported operating system individually and no longer include built-in libraries. This change reduces the tarball download size and increases their security by simplifying updates for required dependencies. 
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
```
Comment (radoslawszulgo) [inline docs/release_notes/5.0.28-24.md:30]:
```suggestion
## Resolved issues
The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
```

File: docs/release_notes/5.0.28-24.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The enhanced telemetry feature provides comprehensive information about how it works, its components and metrics as well as updated methods how to disable telemetry. Read more in [Telemetry on Percona Server for MongoDB](../telemetry.md)
+
+### Reduce mean time to resolve (MTTR) compromised encryption key incidents in KMIP
+
+Starting with this release, Percona Server for MongoDB automatically activates the master encryption keys at startup and periodically checks (polls) their status in a KMIP server. If a master encryption key for a node is not in the Active state, the node reports an error and shuts down. This method allows security engineers to quickly identify which nodes require out-of-schedule master key rotation, such as in the case of compromised keys, without needing to rotate keys for the entire cluster. 
+
+The following diagram provides the sample master key lifecycle with key state polling:
+
+![image](../_images/kmip-state-polling.jpg)
```
Comment (sandraromanchenko) [inline docs/release_notes/5.0.28-24.md]: @radoslawszulgo Isn't it too much for release notes? IMHO, it'll better work on respective feature page if user wants to learn more about this new feature.

Comment (radoslawszulgo) [reply docs/release_notes/5.0.28-24.md]: Too much? Maybe. Let's see. If this attracts at least 1 person it's good. I'll try to lookup if there were some clicks to learn more about KMIP from the Release Notes page. 

The problem is that the image can't be opened in wider screen or with ability to zoom in. It's currently hard to read.

**Review by sandraromanchenko — APPROVED** [review]

**Review by sandraromanchenko — APPROVED** [review]

File: docs/release_notes/5.0.28-24.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+This release of Percona Server for MongoDB includes the following features and improvements:
+
+### Enhanced Telemetry for better product usage reporting
+
+The enhanced telemetry feature provides comprehensive information about how it works, its components and metrics as well as updated methods how to disable telemetry. Read more in [Telemetry on Percona Server for MongoDB](../telemetry.md)
+
+### Reduce mean time to resolve (MTTR) compromised encryption key incidents in KMIP
+
+Starting with this release, Percona Server for MongoDB automatically activates the master encryption keys at startup and periodically checks (polls) their status in a KMIP server. If a master encryption key for a node is not in the Active state, the node reports an error and shuts down. This method allows security engineers to quickly identify which nodes require out-of-schedule master key rotation, such as in the case of compromised keys, without needing to rotate keys for the entire cluster.
```
Comment (igorsol) [inline docs/release_notes/5.0.28-24.md]: This seems to be description from v8.0 but versions prior to v8.0 have different behavior. Let's check with @ktrushin

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/release_notes/5.0.28-24.md (hunk 3 above)
Comment (ktrushin) [reply docs/release_notes/5.0.28-24.md]: The above text is almost correct if we are speaking about _new_ keys. I would add some clarifications:
- `activates the master encryption keys` -> ` activates all new master encryption keys`
- `is not in the Active state` -> `transitions from the Active to another state`
- add the following `If Percona Server for MongoDB detects at startup that data directory is encrypted with an existing key in the Pre-Active state, it logs a warning and continues as usual. An operations team is encouraged to transition the keys to Active state using master key rotation or manually.`

**Review by ktrushin — APPROVED** [review]

**Pull request merged**